examples/blog/README.md

```diff
@@ -3,148 +3,312 @@
 ### Dispatch
 You will need a dispatch cluster deployed and configured, please follow the quick start instruction at [dispatch](https://github.com/vmware/dispatch) repository.
 
-You will need the url of your dispatch installation.
+For minikube deployment, keep a note of the port of your dispatch host and dispatch api-gateway host. This is printed
+out at the end of the dispatch installation, but you can easily fetch it via kubectl:
+
 ```
-DISPATCH_HOST=<your-dispatch-host>
-DISPATCH_API_HOST=<your-dispatch-api-host>
+$ kubectl -n kong get service api-gateway-kongproxy
+NAME                    TYPE       CLUSTER-IP    EXTERNAL-IP   PORT(S)                      AGE
+api-gateway-kongproxy   NodePort   10.101.3.80   <none>        80:32521/TCP,443:32696/TCP   49m
+
+$ export DISPATCH_HOST=$(minikube ip)
+$ export DISPATCH_API_URL=https://$DISPATCH_HOST:32696
 ```
 
-For minikube deployment, keep a note of the port of your dispatch host and dispatch api-gateway host.
+> **Note:** We are setting the `DISPATCH_API_URL` to the host IP and the **https** port.
 
 ### Minio
 
 Minio is a S3 compatible object store, we use it to store blog posts. You can deploy a minio server locally or use S3 instead.
 
 Here we briefly provide a simple way to deploy minio in a kubernetes cluster with helm, if you have deployed dispatch, you should already have a kubernetes cluter and helm ready.
 ```
-helm upgrade minio --install --namespace minio --set serviceType=NodePort stable/minio
+$ export MINIO_ACCESS_KEY=blogaccess
+$ export MINIO_SECRET_KEY=blogsecret
+
+$ helm install --name minio --namespace minio --set serviceType=NodePort,accessKey=$MINIO_ACCESS_KEY,secretKey=$MINIO_SECRET_KEY stable/minio
 ```
 You need to keep a note of minio credentials, including ``hostname``, ``port``, ``accessKey``, ``secretKey``.
 
-For minio deployed in kubernetes:
-
-``hostname`` will be your kubernetes cluster hostname,
+The minio ``hostname`` is simply `DISPATCH_HOST` if you installed into the same kubernetes cluster:
+```
+$ export MINIO_HOST=$DISPATCH_HOST
+```
 
-get ``port`` with
-```kubectl get svc -n minio```,
+To fetch the ``port`` associated with the minio service:
+```
+$ kubectl get svc -n minio
+NAME              TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+minio-minio-svc   NodePort   10.102.89.197   <none>        9000:31390/TCP   37s
+$ export MINIO_PORT=31390
```

Comment:
Also:
```kubectl get svc -n minio -o json | kubectl get svc -n minio -o json | jq '.items[0].spec.ports[0].nodePort'```

Reply:
ok... I actually was going to put that in there.  I'll do it